ui/apps/platform/src/Containers/Audit/ListeningEndpoints/ListeningEndpointsTable.tsx

```diff
@@ -16,7 +16,7 @@ function EmbeddedTable({
     listeningEndpoints: ProcessListeningOnPort[];
 }) {
     return (
-        <TableComposable isNested>
+        <TableComposable isNested aria-label="Listening endpoints for deployment">
```

Comment:
Can you search online for pro and con of `aria-label` versus visible text on page to describe table?

Reply:
Aha, main table below is where the question is even more relevant.

Reply:
I haven't been able to find a good definitive recommendation on this. To me it seems that visible text would be overkill for sighted users, but the `aria-label` would help orient users using a screen reader as they move from table to table through nested contexts.

It would be nice to test out a screen reader here and see how this actually sounds when navigating the page to be sure though.

Our usage throughout the app is mixed - there are quite a few places we use `aria-label` on tables, but definitely not on all of them.